GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -176,279 +176,264 @@ public void DehydrateShouldFailOnWrongDiskLayoutVersion()
 
             this.DehydrateShouldSucceed(
-                new[] { $"{folderToDehydrate1} {FolderDehydrateSuccessfulMessage}", $"Cannot dehydrate folder '{folderToDehydrate2}': '{folderToDehydrate2}' does not exist." },
+                new[] { $"{parentFolderToDehydrate} {FolderDehydrateSuccessfulMessage}", $"Cannot dehydrate folder '{childFolderToDehydrate}': '{childFolderToDehydrate}' does not exist." },
                 confirm: true,
                 noStatus: false,
-                foldersToDehydrate: string.Join(";", folderToDehydrate1, folderToDehydrate2));
+                foldersToDehydrate: string.Join(";", parentFolderToDehydrate, childFolderToDehydrate));
 
             this.Enlistment.UnmountGVFS();
 
-            fileToRead1.ShouldNotExistOnDisk(this.fileSystem);
-            fileToRead2.ShouldNotExistOnDisk(this.fileSystem);
+            fileToReadInParentFolderBackingPath.ShouldNotExistOnDisk(this.fileSystem);
+            fileToReadInChildFolderBackingPath.ShouldNotExistOnDisk(this.fileSystem);
         }
 
-        [TestCase]
+        [TestCase]
         public void FolderDehydrateParentFolderInModifiedPathsShouldOutputMessage()
         {
-            string pathToDelete = this.Enlistment.GetVirtualPathTo("GitCommandsTests");
-            this.fileSystem.DeleteDirectory(pathToDelete);
+            string folderToDelete = "GitCommandsTests";
+            string folderToDeleteVirtualPath = this.Enlistment.GetVirtualPathTo(folderToDelete);
+            this.fileSystem.DeleteDirectory(folderToDeleteVirtualPath);
             GitProcess.Invoke(this.Enlistment.RepoRoot, "reset --hard");
 
-            string folderToDehydrate = Path.Combine("GitCommandsTests", "DeleteFileTests");
+            string folderToDehydrate = Path.Combine(folderToDelete, "DeleteFileTests");
             this.Enlistment.GetVirtualPathTo(folderToDehydrate).ShouldBeADirectory(this.fileSystem);
 
-            this.DehydrateShouldSucceed(new[] { $"Cannot dehydrate folder '{folderToDehydrate}': Must dehydrate parent folder 'GitCommandsTests/'." }, confirm: true, noStatus: false, foldersToDehydrate: folderToDehydrate);
+            this.DehydrateShouldSucceed(new[] { $"Cannot dehydrate folder '{folderToDehydrate}': Must dehydrate parent folder '{folderToDelete}/'." }, confirm: true, noStatus: false, foldersToDehydrate: folderToDehydrate);
         }
 
-        [TestCase]
+        [TestCase]
         public void FolderDehydrateDirtyStatusShouldFail()
         {
-            string fileToCreate = this.Enlistment.GetVirtualPathTo(Path.Combine("GVFS", $"{nameof(this.FolderDehydrateDirtyStatusShouldFail)}.txt"));
-            this.fileSystem.WriteAllText(fileToCreate, "new file contents");
-            fileToCreate.ShouldBeAFile(this.fileSystem);
+            string folderToDehydrate = "GVFS";
+            string fileToCreateVirtualPath = this.Enlistment.GetVirtualPathTo(Path.Combine(folderToDehydrate, $"{nameof(this.FolderDehydrateDirtyStatusShouldFail)}.txt"));

```

Comment:
Minor: since this is getting cleaned up anyway, this line can be simplified to:

`this.Enlistment.GetVirtualPathTo(folderToDehydrate, $"{nameof(this.FolderDehydrateDirtyStatusShouldFail)}.txt");`